tests/gis_tests/gdal_tests/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.gis.db import models
+from django.db import connection
+
+HAS_POSTGIS_2 = (
+    hasattr(connection.ops, 'postgis') and
```

Comment:
is there a reason for using vendor checks instead of the `supports_raster` feature flag?